alts/worker/runners/base.py

```diff
@@ -370,13 +371,14 @@ def publish_artifacts_to_storage(self):
         def write_to_file(file_base_name: str, artifacts_section: dict):
             log_file_path = os.path.join(
                 self._artifacts_dir, f'{file_base_name}_{self._task_id}.log')
-            with open(log_file_path, 'w+t') as f:
-                f.write(f'Exit code: {artifacts_section["exit_code"]}\n')
-                f.write(f'Stdout:\n\n')
-                f.write(artifacts_section['stdout'])
+            with open(log_file_path, 'wb') as file_:
```

Comment:
just `fd` instead `file_`. I guess it's more suitable and short. fd - file descriptor